docs/src/test-reporters-js.md

```diff
@@ -18,50 +17,44 @@ For more control, you can specify reporters programmatically in the [configurati
 
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 const config = {
-  reporter: 'line',
+  reporter: "line",
 };
 
 module.exports = config;
 ```
 
 ```js tab=js-ts
 // playwright.config.ts
-import type { PlaywrightTestConfig } from '@playwright/test';
+import type { PlaywrightTestConfig } from "@playwright/test";
 
 const config: PlaywrightTestConfig = {
-  reporter: 'line',
+  reporter: "line",
 };
 export default config;
 ```
 
 ### Multiple reporters
 
-You can use multiple reporters at the same time. For example  you can use`'list'` for nice terminal output and `'json'` to get a comprehensive json file with the test results.
+You can use multiple reporters at the same time. For example you can use `'list'` for nice terminal output and `'json'` to get a comprehensive json file with the test results.
 
 ```js tab=js-js
 // playwright.config.js
 // @ts-check
 
 /** @type {import('@playwright/test').PlaywrightTestConfig} */
 const config = {
-  reporter: [
-    ['list'],
-    ['json', {  outputFile: 'test-results.json' }]
-  ],
+  reporter: [["list"], ["json", { outputFile: "test-results.json" }]],
 };
 
 module.exports = config;
 ```
 
 ```js tab=js-ts
 // playwright.config.ts
-import type { PlaywrightTestConfig } from '@playwright/test';
+import type { PlaywrightTestConfig } from "@playwright/test";
```

Comment:
There is a lot of formatting in this PR, but I think most of it does not fit Playwright coding conventions. The codebase and most of the examples are using single quotes, for example.

Reply:
Thanks for catching it. My formatter had different settings. Fixed the docs.